### ISSUE TITLE
Setting up MuJoCo on Google Colaboratory

### ISSUE THREAD

**Issue by Akella17** [openai/mujoco-py#358]
The ipython notebook can be found in this link: https://drive.google.com/file/d/1L-SzAYSNLruiJttNrnSk7W2vex2e9yev/view?usp=sharing

**Comment by mokeam** [comment]
hi, have you been able to fix this?

**Comment by Akella17** [comment]
nope

**Comment by Akella17** [comment]
I think I found the issue. MuJoCo's license is based on the activation keys generated from the computer and OS (limited to 3 registrations per year). This practically makes it impossible to use MuJoCo with Colabs.
Hope this helps!!

**Comment by 10jinn01** [comment]
Get an institutional license.

**Comment by KodinGuy** [comment]
Another solution would be to setup a remote server between colab and a licensed pc.

**Comment by KodinGuy** [comment]
Using a rpc server and client.

**Comment by BuildingAtom** [comment]
In case anyone else needs mujoco-py (say for some OpenAI Gym environment), I did get it working with latest version of MuJoCo since it doesn't need a hardware key. I put the code block I wrangled together into a gist here: https://gist.github.com/BuildingAtom/3119ac9c595324c8001a7454f23bf8c8

I had to use MuJoCo on another project and took the time to figure this out issue for Colab. Hopefully this helps some others in the same boat!
